app/assets/stylesheets/_shame.scss

```diff
@@ -989,36 +903,10 @@ tbody th {
   padding: $gutter-half $gutter;
 }
 
-
-.hidden,
-.js-enabled .js-hidden {
-  display: none;
-  visibility: hidden;
-}
-
 .highlight {
   background-color: $yellow;
 }
 
-
-.field_with_errors,
```

Comment:
can still see these classes used in a lot of places in `js`. are you confident it does not break existing functionality?

Reply:
This class is also duplicated within _forms.scss and it has been visually test by @Dan

Reply:
👍 great, ta